chunky/src/java/se/llbit/chunky/ui/render/tabs/AdvancedTab.java

```diff
@@ -101,6 +102,16 @@ public void initialize(URL location, ResourceBundle resources) {
     rayDepth.setRange(1, 25);
     rayDepth.clampMin();
     rayDepth.onValueChange(value -> scene.setRayDepth(value));
+
+    branchCount.setName("Branch count");
+    branchCount.setTooltip("Sets the number of rays cast after the first intersection.");
```

Comment:
This description doesn't mention the reuse of initial rays and the fact that this could affect the quality of low spp renders.

Reply:
Is `branchCount` a good name?

Reply:
Well, I was thinking of it as the number of times the ray branches at its first intersection. Do you have a better name in mind?

And I will try to edit the description to be more complete.